platform/kube/controller.go

```diff
@@ -33,6 +33,9 @@ import (
 	"istio.io/pilot/model"
 )
 
+// NodeAZLabel is the well-known label for kubernetes node AZs
+const NodeAZLabel = "failure-domain.beta.kubernetes.io/zone"
+
```

Comment:
Essentially you can flatten region and zone into one string since Envoy does not have support for regions